jenkins/test-history/buckets.json

```diff
@@ -0,0 +1,11 @@
+{
+    "gs://kubernetes-jenkins/logs/": {
```

Comment:
I dunno, I think we'll want room for richer metadata to describe buckets.  I can't think of a pressing concrete example, but maybe something like owners, contact info, etc.  Either that or we come up with an enforceable naming scheme, which seems like the wrong place to stuff in more info